backend/src/controllers/v3/secretsController.ts

```diff
@@ -467,20 +459,13 @@ export const deleteSecretByNameRaw = async (req: Request, res: Response) => {
 export const getSecrets = async (req: Request, res: Response) => {
   const validatedData = await validateRequest(reqValidator.GetSecretsV3, req);
   const {
-    query: { environment, workspaceId, include_imports: includeImports, folderId }
```

Comment:
I see that we are removing `folderId` in some of these routes, why is that?

Reply:
Its not used anymore so removed it.